docs/src/components/code-block/actions.tsx

```diff
@@ -45,10 +76,19 @@ export const CodeSandboxAction: React.FC<CodeSandboxActionProps> = props => {
       example={props.code}
       examplePath="does/not/do/anything/but/is/required.tsx"
       dependencies={{
-        '@babel/runtime': 'latest'
+        '@pluralsight/ps-design-system-core': 'latest',
+        '@pluralsight/ps-design-system-featureflags': 'latest',
+        '@pluralsight/ps-design-system-normalize': 'latest',
+        '@pluralsight/ps-design-system-theme': 'latest',
+        '@babel/runtime': 'latest',
```

Comment:
why's this one needed?

Reply:
babel runtime was there previously. codesandbox uses it for typescripty stuff i think